src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img"></div>
+
+      <p class="card__text card__text--head">
```

Comment:
Headings should be used for product titles for better semantic structure and SEO. Consider using `<h1>` to `<h6>` tags instead of `<p>` tags.

Reply:
Same - fix it as suggested:

![image](https://github.com/user-attachments/assets/c8a3bde1-fa45-4923-a13a-43dd16c3bd09)

You can use h2 here - h1 would be the page title